### DISCUSSION TITLE
hi could you do this please?

### DISCUSSION THREAD

**Discussion by frivclassic2022** [Tsyuay/FrivReborn#30] [Q&A] [answered]
<img width="1571" alt="Sem título" src="https://user-images.githubusercontent.com/117296290/210181925-fbfe38bc-0b11-4f2e-b5cb-38cf209ddead.png">
please!!!

**Comment by frivclassic2022** [comment]
hello?

**Comment by Tsyuay** [comment, marked as answer]
done
